sequence/sequence_generator.hpp

```diff
@@ -94,17 +94,18 @@ class SeqGen {
     }
 
     /**
-     * Generate sequences such that every pair of sequences have approximately the same edit
-     * distance.
+     * Generate sequences with mutation rate uniformly drawn from [0,1] in order
```

Comment:
I would add a new method, and keep the old one. The old one allows us to control how TS and TSS perform under specific mutation rates. We already saw using the old method that TS/TSS need a mutation rate of >0.1 to perform well, so I think this is worth keeping.